cmd/habitat/community/consensus/raft/helpers.go

```diff
@@ -40,3 +43,23 @@ func getCommunityAddress(communityID string) string {
 func getCommunityRaftDirectory(communityID string) string {
 	return filepath.Join(compass.CommunitiesPath(), communityID, "raft")
 }
+
+func getClusterProtocol(communityID string) protocol.ID {
+	return protocol.ID(filepath.Join("/habitat-raft", "0.0.1", communityID))
+}
+
+func getPublicMultiaddr() (ma.Multiaddr, error) {
```

Comment:
perhaps make this part of compass?

Reply:
Sounds good. Gonna split up this PR because its already way to big, so I think this change will go into the next one